PUBLISHING.md

```diff
@@ -4,87 +4,99 @@
+- [ ] Approved the updated `CHANGELOG.md` file with the @holmesworcester.
+- [ ] Reviewed the [Quiet Planning Board](https://github.com/orgs/TryQuiet/projects/3) and ensured all issues contained in the release candidate are in the `Ready for QA` column.
 
-## Release candidates (alpha releases)
-Pre-release builds should only be triggered from the release branch and then delivered to QA.  
-Alpha versions should be delivered to QA by updating GitHub with a link to the correct versions (for desktop) and numbers of current versions (for mobile). All GitHub issues included in the released alpha version should be moved to the Ready for QA column.
-If QA reports problems that needs to be solved, the fixes must be merged into the release and develop branches, then a patched pre-release is to be built.
+## Preparing a Release Candidate (Alpha)
 
+Alpha releases are pre-release versions of the release which are delivered to QA for testing. They are versioned with a pre-release version number, e.g. `2.1.0-alpha.0`.
+
+1. Generate a [Github Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#) and set it as `GH_TOKEN` environment variable (or include it in the command as shown below).
+1. Trigger a pre-release with `GH_TOKEN=<token> npm run publish --release=[prepatch|preminor|premajor|<EXACT_VERSION>]` (e.g. to create a prerelease of a minor update `npm run publish --release=preminor`) This will increment the versions of every package that has changed, create a release on the [Releases Page](https://github.com/TryQuiet/quiet/releases), and trigger Github Actions to deploy the alpha release to the Google Play Store and App Store.
+1. Manually update the release notes on the [Releases Page](https://github.com/TryQuiet/quiet/releases) with the changes included in the alpha release. See [RELEASE_NOTES_GUIDE.md](RELEASE_NOTES_GUIDE.md) for guidance on writing release notes.
+1. Promote the alpha release on the [Google Play Console](https://play.google.com/console/) to a closed testing track. Contact @holmesworcester if you need access to the organization.
+1. Notify QA that the alpha release is ready for testing.
+
+## Checklist for Production Release
 
-## Pre-release checklist
 - [ ] Build is working correctly, passes automated tests and self-QA
-- [ ] Release candidate is delivered for QA
+- [ ] Alpha was delivered for QA
 - [ ] Sprint column is free from QA reported blocking issues
 - [ ] QA approved the release
-- [ ] All hotfixes to previous releases have been merged into the release (and develop) branch
-- [ ] CHANGELOG.md is up to date
+- [ ] All hotfixes for issues discovered in alpha releases have been merged into the release (and develop) branch
```

Comment:
I don't think this line is equivalent to the one that it replaces. In some cases we'll make full releases (not alpha releases) that include a hotfix to a previous release. We then need to make sure these hotfixes are brought over to the new release because it's possible someone forgot to add them to both the hotfix branch and develop at the time they were created. Does this make sense?

Reply:
I don't think that entirely makes sense as a rule because there may be cases where hotfixes applied to a previous release are solved in more robust way through a refactor in the develop branch, and in that case, it wouldn't make sense to cherry pick the hotfix applied to an old release. To me, it seems better to just focus on ensuring that at the time of a release (alpha or production) merges the appropriate commits from the release branch to `develop`. Otherwise, there becomes this question of how many previous releases back will we comb through every commit in that release branch to check if it was cherry picked to `develop` or not, and if that change should be applied to the current release or if it was fixed in another way.